.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+checks:
+  file-lines:
+    enabled: false
```

Comment:
I honestly wish this wasn't disabled... I like to code on a fairly small laptop and it's been a bit of a pain.

Reply:
I'm in agreement. I'll gladly remove this entry, I know our code severely fails on this one though.

@clintoncwolfe whats your thought?

Reply:
Well, philosophically, I'm the "I want to build features and fix bugs on a product, not deal with linting issues constantly", so I don't care if the files are big, etc. We have editors, we have scrolling mice, we have search.

On new work, I'm all for short files, why not; but I don't see much gain from reducing file length in isolation from other refactorings (I'm all for splitting lib/inspec/cli.rb, for example, but it's not file-length that motivates me).

That is entirely an opinion, though.

Reply:
Lets leave it disabled for now =)

Reply:
Oh. Derp. I assumed it was about line width, not file length.